pages/1.13/administering-clusters/deploying-a-local-dcos-universe/index.md

```diff
@@ -1,38 +1,39 @@
 ---
 layout: layout.pug
-navigationTitle:  Deploying a Local Universe
-title: Deploying a Local Universe
+navigationTitle:  Deploying a Local Catalog
```

Comment:
This is still called local universe. Local catalog has no meaning.

Reply:
Hmm. Will this be confusing for our users? If we use "Catalog" throughout the rest of the docs, and suddenly spring "local universe" on them, how will they know what we're talking about?

Reply:
@somikbehera can provide more guidance